database/src/adapters/genesis.rs

```diff
@@ -0,0 +1,59 @@
+use actix_diesel::Database;
+use diesel::PgConnection;
+use near_chain_configs::{Genesis, GenesisValidationMode};
+
+use crate::adapters::access_keys::store_access_keys_from_genesis;
+use crate::adapters::accounts::store_accounts_from_genesis;
+
+pub async fn store_genesis_records(
+    pool: &Database<PgConnection>,
+    genesis_file_path: String,
+) -> anyhow::Result<()> {
+    let mut accounts_to_store: Vec<crate::models::accounts::Account> = vec![];
+    let mut access_keys_to_store: Vec<crate::models::access_keys::AccessKey> = vec![];
+
+    let genesis = Genesis::from_file(genesis_file_path, GenesisValidationMode::Full);
+
+    let genesis_height = genesis.config.genesis_height;
+
+    for record in genesis.records.0 {
+        if accounts_to_store.len() == 5_000 {
```

Comment:
Why it should be exactly 5000?

Reply:
This figure was found experimentally for the bulk size

Reply:
Why do we ignore the data if the chunk is smaller? What if it's the last chunk?

Reply:
We don't ignore it, accounts/access keys are also stored after looping through all genesis records (L55/56)